app/src/main/java/org/oppia/app/onboarding/OnBoardingActivity.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.onboarding
+
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.profile.ProfileActivity
+import javax.inject.Inject
+
+/** An activity that shows a temporary on_boarding screen until the app is on-boarded then navigates to [ProfileActivity]. */
```

Comment:
Update this to `on-boarding `

Reply:
Just an update: change `on_boarding` to `on-boarding`. Also, I think me might go back `onboarding` word here and everywhere else. But let's wait for this PR and #558 .

Reply:
Done